trio/_tests/test_socket.py

```diff
@@ -1108,7 +1111,8 @@ async def receiver() -> None:
 async def test_many_sockets() -> None:
     total = 5000  # Must be more than MAX_AFD_GROUP_SIZE
     sockets = []
-    for x in range(total // 2):
+    x = 0
+    for x in range(total // 2):  # noqa: B007  # Used for check below
```

Comment:
> B007: Loop control variable not used within the loop body. If this is intended, start the name with an underscore.

can rename it instead of silencing the warning. Though I'm used to underscore-variables meaning they're unused so I'm not loving the convention, but /shrug

Reply:
Changed in https://github.com/python-trio/trio/commit/11479806a721b0e27e6d4099eab91f59db247976